Planetary.xcworkspace/contents.xcworkspacedata

```diff
@@ -16,6 +16,9 @@
    <FileRef
       location = "group:Secrets">
    </FileRef>
+   <FileRef
+      location = "group:Support">
```

Comment:
This fell through the cracks probably after switching to the Support branch. It seems to be harmless anyway.

Reply:
Oops, good catch. I removed this change.